Automattic/wp-calypso
# Add monthly pricing test

#### Changes proposed in this Pull Request

* Add the monthly pricing test to Calypso. This PR requires the server-side implementations.

#### Testing instructions

💡  For full and **highly-detailed test instructions**, please see: pau2Xa-2ak-p2

-- or --



* Make sure you're proxied.
* Sandbox `public-api.wordpress.com` which D51146-code is applied to.
* (added) D51640-code also needs to be placed on the sandbox.
* ~~Assign yourself to a test group of `monthly_pricing_test_phase_1` by following the instruction ( pau2Xa-251-p2#comment-7999 )~~
* Assigny yourself to the `treatment` group of `monthlyPricing` a/b test.
* Start the process of creating a new website with the `onboarding` flow( `/start/user` ).
* If you're in the `treatment` test group, the plans page should show monthly dotcom plans as below:
  <img width="893" alt="Create_a_site_—_WordPress_com" src="https://user-images.githubusercontent.com/212034/96158436-7b928e00-0f4e-11eb-9e50-06311af0c325.png">
* Pick a plan except for the free plan.
* Your site is newly created and you will see the checkout page that shows what you get with annual/biennial plans.
  <img width="811" alt="checkout_e28094_wordpress_com-2" src="https://user-images.githubusercontent.com/212034/96991514-81e2c480-1563-11eb-9cd4-c5dc52b73651.png">
  <img width="811" alt="checkout_e28094_wordpress_com-3" src="https://user-images.githubusercontent.com/212034/96991532-8909d280-1563-11eb-89bd-b071504861ab.png">
